client/rpc/status.go

```diff
@@ -63,17 +64,17 @@ func StatusCommand() *cobra.Command {
 				},
 			}
 
-			output, err := clientCtx.LegacyAmino.MarshalJSON(statusWithPk)
+			output, err := json.Marshal(statusWithPk)
```

Comment:
We cannot use `json.Marshal` as the pubkey will not display properly:

```json
"PubKey":{"type":"tendermint/PubKeyEd25519","value":"zYJRQ/WvnTy4u7P/nYTLSMHw2xdzjrga7wsYRFVAjrg="},
```

vs

```json
"pub_key":{"key":"zYJRQ/WvnTy4u7P/nYTLSMHw2xdzjrga7wsYRFVAjrg="}
```

Reply:
Do you think it is appropriate to implement the "MarshalJSON() ([]byte, error)" method of "pubkey"

Reply:
Maybe use [tmjson](https://github.com/cometbft/cometbft/blob/c3acdb866f885e94dcd0e1ef8ecce53ff689216c/libs/json/encoder.go#L22)